R/learner_aorsf_surv_aorsf.R

```diff
@@ -173,7 +173,11 @@ LearnerSurvAorsf = R6Class("LearnerSurvAorsf",
         pred_type = "surv",
         .args = pv
       )
-      mlr3proba::.surv_return(times = utime, surv = surv)
+      diffs_time = c(utime[1], diff(utime))
```

Comment:
If this could be done via a function in `aorsf` is that acceptable @sebffischer?